app/views/shared/fields/_access_control_level.html.erb

```diff
@@ -7,11 +7,13 @@
 	if accessGroup != nil
 
 		if accessGroup.include?('public')
-		  viewAccess = nil
-	  elsif accessGroup.include?('local')
+		    viewAccess = nil
```

Comment:
what if instead of `nil` we called this `public`, or even make it a symbol? It seems like that's what nil is representing here.

Then below for conditonals you could say `if viewAccess != :public`, for example, instead of a nil check. There may be other downstream issues with this, just a thought.

Reply:
@VivianChu I have no idea how nil is used here but I believe the purpose is just not to show the label for the public view. So I think either way will be fine.

Reply:
@lsitu - I used the symbol way that Matt suggested and the tests still passed locally.